backend/ed_platform/views.py

```diff
@@ -268,6 +315,40 @@ def get_workshops():
                          models.Workshop.query.all()))
     return jsonify({"workshops": workshops})
 
+@app.route('/api/upcoming-workshops')
+def get_upcoming_workshops():
+    trimmed_workshop_list = []
+    date_today = datetime.datetime.now()
+    date_today - date_today.replace(tzinfo=None)
+    workshop_data = list(map(lambda t: workshop_schema.dump(t).data,
+                         models.Workshop.query.all()))
+    for workshop in workshop_data['workshops']:
+        if (len(workshop['sessions']) > 0):
+            id = workshop['id']
+            title_str = workshop['title']
+            display_name_str = workshop['instructor']['display_name']
+            session_date = parser.parse(workshop['sessions'][0]['date_time'])
+            session_date = session_date.replace(tzinfo=None)
+            workshop_dict = {
```

Comment:
We are using Marshmallow Schema files to generate Json.  You don't need to create these dictionaries manually.  If we consistently use the Schema files then we produce consistent Json structures for the front end.  I would create a list of Workshop models that meet the requirements, then covert that list of workshops to json with a line like:

models.WorkshopAPISchema().jsonify(**MyWorkshopList**,many=True)